apps/shorturl/kinds/shorturl.cue

```diff
@@ -0,0 +1,14 @@
+package kinds
+
+shorturl: {
+	kind:		"ShortURL"
+	pluralName:	"ShortURLs"
+	schema: {
+		spec: {
+			path: string
+			uid: string
+			lastSeenAt: int64
+			shortURL: string
+		}
```

Comment:
UID will be the k8s name

```suggestion
		spec: {
			path: string
		}
		status: {
			lastSeenAt: int64
		}
```

lastSeenAt seems more appropriate for status.

Reply:
I put the `uid` here because it is being used as part of the `shortURL` and I don't have access to the resource name at creation time (in the mutator) to build the shortURL and return it.
What is the difference between `status` and `spec`? I understand the `status` is reserved for calculated fields.
We have business logic around the `lastSeenAt` where we have a background process that deletes the resources that have not been used, so we update this value everytime a shortURL is used.